queries.sql

```diff
@@ -1,3 +1,8 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon' ;
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31' ;
+SELECT name FROM animals WHERE neutered =true AND escape_attempts<3 ;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg>10.5 ;
+SELECT * FROM animals WHERE neutered =true ;
```

Comment:
- [ ] Please write the queries for the [last two quests](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#:~:text=Find%20all%20animals%20not,precisely%2010.4kg%20or%2017.3kg).